test-lib.sh

```diff
@@ -27,6 +27,7 @@ EXPECTED_EXIT_CODE=0
 function ct_cleanup() {
   ct_show_resources
   for cid_file in "$CID_FILE_DIR"/* ; do
+    [ -f "$cid_file" ] || break
```

Comment:
If dir `$CID_FILE_DIR` contains more files then only first is checked. What about `continue`?

Reply:
good idea.

Reply:
fixed.